test/Interop/Cxx/members/emit-method-from-called-method.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+
+import MethodCallsMethod
+import StdlibUnittest
+
+var MembersTestSuite = TestSuite("MembersTestSuite")
+
+MembersTestSuite.test("emit-method-from-called-method") {
+  expectEqual(42, callMethod(41))
```

Comment:
How about tests where we call the methods/constructors from Swift, not just where we call a C++ function that does that?

Reply:
Does this currently run? This should work in Swift:
```
expectEqual(42, Incrementor().callIncrement(41))
```

Reply:
While we could do that, I think it's out of scope for this PR, as the issue we have at hand is missing IR definition for an inline function when called from a member. I don't think that adding the additional tests will provide more coverage for that issue.

Reply:
OK, that's fine, but this is test coverage that we probably should have. I'll submit a follow-up PR once this lands.